docs/core_systems/service_orchestration_dynamic.md

```diff
@@ -1,9 +1,61 @@
 # Service Orchestration (Dynamic)
```

Comment:
As we talked about, the name should be Dynamic Service Orchestration (and in the menu too)